server/chat-plugins/daily-spotlight.ts

```diff
@@ -132,8 +132,16 @@ export const pages: Chat.PageTable = {
 				});
 				for (const [i] of keys.entries()) {
 					const html = await renderSpotlight(room.roomid, key, i);
-					buf += `<tr><td>${i ? i : 'Current'}</td><td>${html}</td></tr>`;
-					if (!user.can('announce', null, room)) break;
+					buf += `<tr><td>${i ? i : 'Current'}</td><td>${html}</td>`;
+					if (user.can('announce', null, room)) {
+						const deleteCommand = (
+							i === 0 ?
+								`/msgroom ${room.roomid},/removedaily ${key}` :
+								`/msgroom ${room.roomid},/removedaily ${key}, ${i}`
+						);
+						buf += `<td><button class="button" name="send" value="${deleteCommand}">Delete</button></td>`;
```

Comment:
Okay, so, if I'm understanding correctly, the first one deletes all spotlights, but the second onward only deletes one of the spotlights? But they're all labeled "Delete"? That seems wrong to me.

Reply:
So it's just that the help text for `/removedaily` is wrong.